src/App.js

```diff
@@ -1,18 +1,32 @@
 import './App.css';
-import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { useGameContext } from './context/GameContext';
 
 function App() {
-  const [deck, setDeck] = useState(initialCards);
-  const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
-  const [playerTwoHand, setPlayerTwoHand] = useState([]);
-  const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+  // const [deck, setDeck] = useState(initialCards);
+  // const [playerOneHand, setPlayerOneHand] = useState([]);
+  // const [selectedCard, setSelectedCard] = useState();
+  // const [playerTwoHand, setPlayerTwoHand] = useState([]);
+  // const [playerThreeHand, setPlayerThreeHand] = useState([]);
+  // const [from, setFrom] = useState('deck');
+  // const [to, setTo] = useState(1);
+
+  const {
+    deck,
+    setDeck,
+    playerOneHand,
+    setPlayerOneHand,
+    selectedCard,
+    setSelectedCard,
+    playerTwoHand,
+    setPlayerTwoHand,
+    playerThreeHand,
+    setPlayerThreeHand,
+    from,
+    to,
+  } = useGameContext();
 
   function findCardIndex(value, suit, cards) {
     return cards.findIndex((card) => card.value === value && card.suit === suit);
```

Comment:
This function can be placed into the exectuePassButton so you're not prop drilling down extra props.